uportal-war/src/main/java/org/jasig/portal/shell/PortalShellBuildHelperImpl.java

```diff
@@ -367,4 +387,21 @@ private ISchemaExport getSchemaExport(String persistenceUnit) {
         
         throw new IllegalArgumentException("No ISchemaExport bean found for persistence unit: '" + persistenceUnit + "'");
     }
+
+    @Override
+    public String getFilesListFromFile(String filePathsFile) throws IOException {
+        String filepaths = "";
```

Comment:
Minor but since you are doing a lot of string concatenation it would be more efficient to use StringBuffer instead of String.